source/fundamentals/crud/read-operations/watch.txt

```diff
@@ -0,0 +1,235 @@
+   if err != nil {
+      panic(err)
+   }
+   defer changeStream.Close(context.TODO())
+
+   // iterate over the cursor to print the change stream events
+   for changeStream.Next(context.TODO()) {
+      fmt.Println(changeStream.Current)
+   }
+
+If you modify the ``tea.ratings`` collection in a separate shell, this code will print
+your changes. Inserting a document with a ``name`` value of "``White Peony``" and a 
+``rating`` value of ``4`` will output the following change event:
+   
+.. code-block:: go
+   :copyable: false
+
+   map[_id:map[_data:8262DFFF73000000262B022C0100296E5A1004FB2444E03F9A4C5A
+   B3C7E868E387116846645F6964006462DFFF538BC04F3816B8BCD10004] clusterTime:
+   {1658847091 38} documentKey:map[_id:ObjectID("62dfff538bc04f3816b8bcd1")] 
+   fullDocument:map[_id:ObjectID("62dfff538bc04f3816b8bcd1") 
+   rating:4 type:White Peony] ns:map[coll:ratings db:tea] operationType:insert]
+
+.. note::
+
+   The ``_data``, ``clusterTime``, and ``ObjectID`` values in the output will 
+   vary with each user.
+
+Pipeline Parameter
+~~~~~~~~~~~~~~~~~~
+
+Use a pipeline parameter to modify the change stream output. This parameter must be 
+a ``non-nil`` value, but you can pass an empty aggregation pipeline to return an unfiltered
+change stream. Format the pipeline parameter as an array of documents, with each 
+document representing an aggregation stage. 
+
+You can use the following pipeline stages in this parameter:
+
+- ``$addFields``
+- ``$match``
+- ``$project``
+- ``$replaceRoot``
+- ``$replaceWith``
+- ``$redact``
+- ``$set``
+- ``$unset``
+
+Example
+```````
+
+The following example opens a change stream on the ``tea`` database but only watches for
+new delete operations:
+
+.. code-block:: go
+
+   db := client.Database("tea")
+
+   pipeline := bson.D{{"$match", bson.D{{"operationType", "delete"}}}}
+   changeStream, err := db.Watch(context.TODO(), mongo.Pipeline{pipeline})
+   if err != nil {
+      panic(err)
+   }
+   defer changeStream.Close(context.TODO())
+
+   for changeStream.Next(context.TODO()) {
+      fmt.Println(changeStream.Current)
+   }
+
+If you delete the ``tea.ratings`` document with a ``name`` value of 
+"``White Peony``" in a separate shell, this code will output the following:
+
+.. code-block:: go
+   :copyable: false
+   
+   {"_id": {"_data": "8262E0051B000000102B022C0100296E5A1004FB2444E03F9A4C5AB
+   3C7E868E387116846645F6964006462DFFF538BC04F3816B8BCD10004"},"operationType":
+   "delete","clusterTime": {"$timestamp":{"t":"1658848539","i":"16"}},"ns": 
+   {"db": "tea","coll": "ratings"},"documentKey": {"_id": {"$oid":"62dfff538bc04f3816b8bcd1"}}}
+
+.. note::
+
+   The ``Watch()`` method was called on the ``tea`` database, so the code outputs
+   new delete operations in any ``tea`` collection.
+
+
+Opts Parameter
+~~~~~~~~~~~~~~
+
+You can use an opts parameter to modify the behavior of the ``Watch()`` method. This
+parameter is optional and can remain ``nil``. 
```

Comment:
Suggestion: "Use the opts parameter to modify the behavior of the ``Watch()`` method."